addons/MobCompass/MobCompass.lua

```diff
@@ -223,10 +223,12 @@ do
             end
             local next45angle = math.ceil((angle+pi/8)/(pi/4))
             if next45angle ~= last45angle then
+			   if next45angle ~= nil and last45angle ~=nil then
                 arrow_map[last45angle]:color(255,255,255)
                 arrow_map[next45angle]:color(255,0,0)
                 last45angle = next45angle
             end
+            end
```

Comment:
Need to fix the indentation here from lines 226 through 230. Please use 4 spaces to indent and no tabs.